console-extensions.json

```diff
@@ -9,5 +9,45 @@
         "kind": "FlashSystemCluster"
       }
     }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "ibmstoragesystem",
+      "insertBefore": "persistentvolumes",
+      "section": "storage",
+      "name": "IBM Flash Systems",
+      "href": "/k8s/all-namespaces/odf.ibm.com~v1alpha1~FlashSystemCluster"
+    },
+    "flags": {
+      "required": ["ODF_IBM_STORAGE"]
+    }
```

Comment:
I am not sure if you need this for testing as well ?
You are already exposing a route for dashboard so it can work out for testing.

Reply:
This will be remove when odf-console is ready for deploy in ocp cluster. But currently we need to keep it to make there is a place for test entry.